common/djangoapps/entitlements/models.py

```diff
@@ -214,11 +215,28 @@ def is_entitlement_redeemable(self):
         return self.policy.is_entitlement_redeemable(self)
 
     def to_dict(self):
-        """ Convert entitlement to dictionary representation. """
+        """
+        Convert entitlement to dictionary representation including relevant policy information.
+
+        Returns:
+            The entitlement UUID
+            The associated course's UUID
+            The date at which the entitlement expired. None if it is still active.
+            The localized string representing the date at which the entitlement expires.
+        """
+        expiration_date = None
+        if self.get_days_until_expiration() < settings.ENTITLEMENT_EXPIRED_ALERT_PERIOD:
+            expiration_date = strftime_localized(
+                datetime.now(tz=pytz.UTC) + timedelta(days=self.get_days_until_expiration()),
```

Comment:
Nit: I think you can do this as [`datetime.utcnow`](https://docs.python.org/2/library/datetime.html#datetime.datetime.utcnow)

Reply:
I thought so too originally but you can't actually. You need datetimes without the +/- associated with them when utilizing timedeltas, so you're out of luck when using .utcnow()

Reply:
lololol thanks for the clarification @jlajoie

Reply:
Yeeeup 😞